widget/embedded/src/components/SwapDetails/SwapDetails.tsx

```diff
@@ -289,6 +292,17 @@ export function SwapDetails(props: SwapDetailsProps) {
                 onClick={handleCopy.bind(null, requestId || '')}>
                 <CopyIcon size={16} color="gray" />
               </IconButton>
+              <IconButton
+                variant="ghost"
+                style={{ padding: '0' }}
+                onClick={() =>
+                  window.open(
+                    `https://scan.rango.exchange/swap/${requestId}`,
+                    '_blank'
+                  )
+                }>
+                <LinkIcon size={16} />
+              </IconButton>
```

Comment:
Can you update skeleton for swap details page? (I think we could add some skeleton for icons as well (two blue squares) because they are not clickable at the time of loading)
Please also add a skeleton for `created at / finished at` part


![Recording 2024-01-07 at 10 07 02](https://github.com/rango-exchange/rango-client/assets/99180934/a7a56cec-e76a-46c0-b723-90bd74bd8a4a)

Reply:
fixed